frontend/server/src/main/java/org/pytorch/serve/wlm/ModelManager.java

```diff
@@ -71,6 +73,29 @@ public ModelArchive registerModel(String url, String defaultModelName)
                 defaultModelName);
     }
 
+    public void registerAndUpdateModel(String modelName, JsonObject modelInfo)
+            throws ModelException, IOException {
+
+        boolean defaultVersion = modelInfo.get("defaultVersion").getAsBoolean();
```

Comment:
Same here. const strings / prop names to either global location or use getter methods.